Plugins/FastScaling/NativeFastScaling/include/fastscaling.h

```diff
@@ -144,16 +144,16 @@ typedef struct ConvolutionKernelStruct {
 typedef struct RenderDetailsStruct {
     //Interpolation and scaling details
     InterpolationDetails * interpolation;
-    //How large the interoplation window needs to be before we even attempt to apply a sharpening
+    //How large the interpolation window needs to be before we even attempt to apply a sharpening
     //percentage to the given filter
     float minimum_sample_window_to_interposharpen;
 
 
-    // If possible to do correctly, halve the image until it is [interpolate_last_percent] times larger than needed. 3 or greater reccomended. Specify -1 to disable halving.
+    // If possible to do correctly, halve the image until it is [interpolate_last_percent] times larger than needed. 3 or greater recommended. Specify -1 to disable halving.
     float interpolate_last_percent;
 
     //The number of pixels (in target canvas coordinates) that it is acceptable to discard for better halving performance
-    float havling_acceptable_pixel_loss;
+    float halving_acceptable_pixel_loss;
```

Comment:
This is an ABI change and impacts other repositories.

Reply:
Why do you think it impacts other repositories?

Reply:
As long as all these instances are changed it should be fine: https://github.com/imazen/resizer/search?q=havling_acceptable_pixel_loss&unscoped_q=havling_acceptable_pixel_loss

Reply:
Because I ran across it when I was working on imageflow -- [imageflow@c6cc524:  c_components/tests/test_integration.cpp](https://github.com/imazen/imageflow/blob/c6cc524fe8f3664452108e140a4d66e050345290/c_components/tests/test_integration.cpp#L183) -- it's just a comment, but it should be cleaned up.

I'm confident I got all of the ones in this repository w/ 6bd0cbe02e72ae2a52312d4032110cc803cbdf6f

That's the only one in the organization outside this repository.
https://github.com/search?q=org%3Aimazen+havling_acceptable_pixel_loss&type=Code